proto/viam/app/v1/app.proto

```diff
@@ -441,6 +444,19 @@ message GetRobotRequest {
   string id = 1;
 }
 
+message GetRoverRentalRobotsRequest {}
```

Comment:
We don;t pass in anything because its harcoded to be robots in Location with name "Rover Rental". Checking on a string is bad practice, but this is the plan for now.